ci/ci_test.sh

```diff
@@ -17,6 +17,13 @@ elif [[ $SELECTED_TEST_SUITE == $"all" ]]; then
   rake test:plugins         # Run all plugins tests
 else
   echo "Running core tests"
-  rake test:install-core    # Install core dependencies for testing.
+  rake plugin:install-core    # Install core dependencies for testing.
```

Comment:
If I understand correctly running the unreleased gem was problematic?
So before merging we will revert to `test:install-core` or this is a permanent change?